tests/richText.js

```diff
@@ -413,8 +413,8 @@ define([
             _.each([
                 ["one two", 3, "one/data/text two"],
                 ["one two", 4, "one /data/text two"],
-                ["one\n\ntwo", 3, "one/data/text\n\ntwo"],
-                ["one\n\ntwo", 4, "one\n/data/text\ntwo"],
+                ["one\n\ntwo", 3, "one/data/text \n\ntwo"],
+                ["one\n\ntwo", 4, "one\n/data/text \ntwo"],
```

Comment:
I always found it a bit odd that the extra space did not show up in these tests. Now it does, although it's inconsistent when editing a label, for example, because the trailing space will eventually get stripped in some cases after selecting the mug a couple times. I believe CKEditor or something in Vellum is collapsing whitespace, which seems like a separate bug.